src/yearToWords/index.js

```diff
@@ -0,0 +1,17 @@
+import { useSelector } from 'react-redux'
+
+import { numberToWords } from '../constans'
+
+export const YearToWords = () => {
+  const year = useSelector(state => state.date)
+
+  const convertToWords = (year) => {
+    return year.toString().split('').map(digit => numberToWords[digit]).join(' ')
+  }
```

Comment:
why do you need function for this?

Reply:
You mean just do <span>{year.toString().split('').map(digit => numberToWords[digit]).join(' ')}</span> ?